src/components/Shared/Countdown.tsx

```diff
@@ -113,23 +116,25 @@ class Countdown extends React.Component<IProps, IState> {
     return (
       <div className={containerClass}>
         <div className={css.percentageContainer}>
-          <div style={{backgroundColor: "blue", height: "2px", width: percentageComplete + "%"}}></div>
+          <div style={{ backgroundColor: "blue", height: "2px", width: percentageComplete + "%" }}></div>
         </div>
         {this.props.detailView ?
-            <span>Proposal ends:</span>
+          <span>Proposal ends:</span>
           : " "
         }
-        <span className={css.timeSection}>
-           <strong>{this.addLeadingZeros(countDown.days)}d</strong>
-        </span>
-        &nbsp;:&nbsp;
+        {
+          Number(countDown.days) ? <span><span className={css.timeSection}><strong>{this.addLeadingZeros(countDown.days)}d</strong></span> &nbsp; : &nbsp;</span> : ""
```

Comment:
should already by a Number, why the conversion?

Reply:
For some reason at the time I was worried it was a string.  I'll remove the 'Number' (after testing, of course).

Reply:
Done